utilities_uv.py

```diff
@@ -276,24 +278,30 @@ def move_island(island, dx, dy):
 	bmesh.update_edit_mesh(me)
 
 
-def rotate_island(island, uv_layer = None, angle = 0, center_x=0, center_y=0):
-	'''Rotate a list of faces by angle (in radians) around a center'''
-
-	if uv_layer is None:
-		me = bpy.context.active_object.data
-		bm = bmesh.from_edit_mesh(me)
-		uv_layer = bm.loops.layers.uv.verify()	
-
+def translate_island(island, uv_layer, delta):
 	for face in island:
 		for loop in face.loops:
-			x, y = loop[uv_layer].uv
-			xt = x - center_x
-			yt = y - center_y
-			xr = (xt * math.cos(angle)) - (yt * math.sin(angle))
-			yr = (xt * math.sin(angle)) + (yt * math.cos(angle))
+			loop[uv_layer].uv += delta
+
+
+def rotate_island(island, uv_layer=None, angle=0, pivot=None):
+    '''Rotate a list of faces by angle (in radians) around a center'''
+    rot_matrix = mathutils.Matrix.Rotation(-angle, 2)

```

Comment:
Unwrap and Sort tools work for me with angle instead of -angle. I'm using your code from master on a "Monkey" mesh.